content/_index.md

```diff
@@ -0,0 +1,169 @@
+        #icon: download
+      # Optionally, add an alternative CTA link
+      # cta_alt:
+      #   label: Our mission
+      #   url: mission
+      # Optionally, add a note under the Call-To-Action button
+      cta_note:
+        label: >-
+                  
+      # Add your Hero text here
+      text: |-
+        Our interactive computing platform gives your community a digital home to create and share knowledge, and a global network of communities to learn from.
+    design:
+      # Choose an optional background color, gradient, image, or video
+      background:
+        text_color_light: true
+        image:
+          filename: "herobg.png"
+          parallax: false
+          brightness: 1
+
+  - block: markdown
+    id: service
+    content:
+      title: We give communities a home in the cloud for interactive computing.
+      subtitle: 
+      text: |
+
+        Our interactive computing platform **empowers communities to design a hub for their unique workflows**. They bring together open source tools, data, and computational resources in a remotely-accessible interactive environment for creating and sharing knowledge.
+
+        {{< figure src="/images/home/hub-diagram.png" width="66%" >}}
+
+                
+        <style>
+          img.logo {
+            padding: 0 3em;
+            margin-top: 1em;
+          }
+        </style>
+
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="section-heading d-flex flex-wrap justify-content-center col-12 mb-3 text-center">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyterlab.svg" alt="Jupyter Lab logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyterhub.svg" alt="Jupyter Hub logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyterbook.svg" alt="Jupyter Book logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyter.svg" alt="Jupyter logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/myst.svg" alt="MyST logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/binder.svg" alt="Binder logo">
+                </div>
+            </div>
+        </div>
+
+        {{< cta cta_text="Learn about our interactive computing platform" cta_link="/service" cta_new_tab="false" >}}
+
+
+
+  - block: markdown
+    id: numbers
+    content:
+      title: We serve a global network of communities
+      subtitle: 
+      text: |
+
+        We serve **over 80 communities across the globe** with **over 7000 active users** that are dedicated to creating and sharing knowledge. This includes **educational communities** that share knowledge within a community, **research communities** that create advances at the frontier of knowledge, and **applied knowledge communities** that use advances in knowledge to serve the public.
```

Comment:
This is the third section of the page that mentions "communities" but the first to specifically mention research and education communities. Should we maybe make that clear earlier on?

Reply:
I'll add another "research and education" shout-out to the section just above